finished-application/frontend/components/styles/NavStyles.js

```diff
@@ -52,6 +52,9 @@ const NavStyles = styled.ul`
       outline: none;
       &:after {
         width: calc(100% - 60px);
+        @media (max-width: 700px) {
```

Comment:
added the mq so the finished product could match the starter file a little closer since it didn't seem too far fetched for these to match a bit closer.